plexapi/audio.py

```diff
@@ -125,6 +132,30 @@ def sync(self, bitrate, client=None, clientId=None, limit=None, title=None):
 
         return myplex.sync(sync_item, client=client, clientId=clientId)
 
+    def sonicallySimilar(
+        self: Self,
+        limit: int = 30,
+        maxDistance: float = 0.25,
+        **kwargs,
+    ) -> "list[Self]":
```

Comment:
```suggestion
    ) -> List[TAudio]:
```

Reply:
Value 'list' is unsubscriptable for py3.8, using List from typing instead